### PR TITLE
fixed Spinner pop up issue on Dialogs

### PR DESCRIPTION
**Pull request by tanoDxyz**
when spinner is shown in layouts that are inflated for Dialogs like Alert dialog...some drop down items were not shown and this is fixed . now spinner behaves as usual on dialogs.

### CODE REVIEW

File: library/src/main/java/org/angmarch/views/NiceSpinner.java [hunk 1]
```diff
@@ -150,19 +149,16 @@ private void init(Context context, AttributeSet attrs) {
         setClickable(true);
 
         backgroundSelector = typedArray.getResourceId(R.styleable.NiceSpinner_backgroundSelector, R.drawable.selector);
+
+
```
Comment (arcadefire) [inline library/src/main/java/org/angmarch/views/NiceSpinner.java]: Can you remove this extra line?

File: library/src/main/java/org/angmarch/views/NiceSpinner.java [hunk 2]
```diff
@@ -123,12 +127,7 @@ public void onRestoreInstanceState(Parcelable savedState) {
             if (bundle.getBoolean(IS_POPUP_SHOWING)) {
                 if (popupWindow != null) {
                     // Post the show request into the looper to avoid bad token exception
-                    post(new Runnable() {
-                        @Override
-                        public void run() {
-                            showDropDown();
-                        }
-                    });
+                    post(this::showDropDown);
```
Comment (arcadefire) [inline library/src/main/java/org/angmarch/views/NiceSpinner.java]: Be sure to add this in the library's `build.gradle` file to enable Java 8 features like passing the method as a reference:
```
compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
}
```

File: library/src/main/java/org/angmarch/views/NiceSpinner.java (hunk 1 above)
Comment (tanoDxyz) [reply library/src/main/java/org/angmarch/views/NiceSpinner.java]: will check it

File: library/src/main/java/org/angmarch/views/NiceSpinner.java (hunk 2 above)
Comment (tanoDxyz) [reply library/src/main/java/org/angmarch/views/NiceSpinner.java]: ok

**Comment by tanoDxyz** [comment]
ok all done mentioned. check it once...

**Comment by asknask** [comment]
Is this really fixed? I'm using nice spinner inside a MaterialDialog. I have 6 items in my data but the spinner only shows 3.

**Comment by tanoDxyz** [comment]
> Is this really fixed? I'm using nice spinner inside a MaterialDialog. I have 6 items in my data but the spinner only shows 3.
> 
> @arcadefire @tanoDxyz Could you check this?

well if the spinner has a new version.

**Comment by pzienowicz** [comment]
Still not working, used the newest version.

**Comment by souragar** [comment]
This is still not fixed

**Comment by tanoDxyz** [comment]
> This is still not fixed

well i fixed the code but as obvious there's no change in version codes since then . so when you add the dependency you are using the old version .

**Comment by tanoDxyz** [comment]
> This is still not fixed

well i fixed the code but as obvious there's no change in version codes since then . so when you add the dependency you are using the old version . and i believe @arcadefire  could fix that.

**Comment by pzienowicz** [comment]
@arcadefire, could you?

**Comment by arcadefire** [comment]
Sorry guys, I had literally zero time to dedicate on this project. I've just drafted a new release: https://github.com/arcadefire/nice-spinner/releases/tag/1.4.4.

**Comment by pzienowicz** [comment]
Thanks @arcadefire, but please make the build passing, because now the tag is not accessible in jitpack:
https://jitpack.io/#arcadefire/nice-spinner

**Pull request merged**